src/composables/FileExplorer.js

```diff
@@ -16,21 +16,22 @@ export function createFolder(id, folder, name) {
 
 /**
  * Create and add a new file.
- * @param {String} id - Absolute path of file.
+ * @param {FileStatus[]} fileInformation - Absolute path and status of a file.
```

Comment:
receiving an array of FileStatus but is named fileInformation, and without an S. Check the description also.

Reply:
or maybe it's just the wrong type...